readme.md

```diff
@@ -32,8 +33,35 @@ a start. It's has the necessary features of a basic text editor and a few fancy
 extras. Those fancy extras stay out of your way until you want them.
 
 
+### Installation (Linux / Mac OS)
+
+* Change directory `cd` to the downloaded directory (or local repository):
+  `ci_edit`.
+
+```
+$ sudo ./install.sh
+```
+
+* **Note:** This script creates a copy of the repository in the directory
+  `/opt/ci_edit`; the **update**, overwrites that copy.
+
+### Usage
+
+* This command opens the text editor from any directory
+
+```
+$ cedt
```

Comment:
What are the advantages for using the name cedt? I suppose we can avoid typing all those pesky 'i's :)

How about a command name like:
- ci
- ciedit
- ci_edit
- edit
- ce

My concern with cedt is that it's an awkward thing to type on a qwerty keyboard. The middle finger hits all three rows, followed by a reach for the 't'.

Trivia: the shortcut I use for ci_edit is 'we' for no other reasons than it's easy and fast to type; and didn't conflict with another command. I'd given some thought to changing the name of ci_edit to we_edit, but thought ci_edit was more distinct when spoken.